tests/providers/amazon/aws/operators/test_batch.py

```diff
@@ -305,40 +337,55 @@ def test_monitor_job_with_logs(
 
 
 class TestBatchCreateComputeEnvironmentOperator:
+    @pytest.fixture(autouse=True)
+    def setup_test_cases(self):
```

Comment:
https://github.com/apache/airflow/blob/main/TESTING.rst#id5

> We are in the process of converting all unit tests to standard "asserts" and pytest fixtures so if you find some tests that are still using classic setUp/tearDown approach or unittest asserts, feel free to convert them to pytest.

Not sure whether we should usefixture in this case

Reply:
> @pytest.fixture(autouse=True)

It is `pytest`'s builtin replacement for xunit-style setup / teardown with additional beneficials, like use another fixtures without some hacks

Use usefixture not always (almost never) detected by IDE's as autosuggestion it really annoying especially if it intends access to class instance attributes. It good ability if you need to assign clear fixture to specific namespaces (classes) in module and can't use:
- `@pytest.fixture(autouse=True, scope="class")`
- `@pytest.fixture(autouse=True, scope="function")`

Reply:
> It is pytest's builtin replacement for xunit-style setup / teardown with additional beneficials, like use another fixtures without some hacks

Yep, this is why I'm not sure whether this is something we want according to the document. I'm actually good with such things, but just want to make sure. If this is what we want, should we update the doc to reduce confusion?